src/win/winapi.h

```diff
@@ -4725,6 +4725,25 @@ typedef DWORD (WINAPI *sPowerRegisterSuspendResumeNotification)
                HANDLE        Recipient,
                _PHPOWERNOTIFY RegistrationHandle);
 
+/* from Winuser.h */
+typedef VOID (CALLBACK* WINEVENTPROC)
+             (HWINEVENTHOOK hWinEventHook,
+              DWORD         event,
+              HWND          hwnd,
+              LONG          idObject,
+              LONG          idChild,
+              DWORD         idEventThread,
+              DWORD         dwmsEventTime);
+
+typedef HWINEVENTHOOK (WINAPI *sSetWinEventHook)
```

Comment:
Curious- why is this needed? [https://msdn.microsoft.com/en-us/library/windows/desktop/dd318453(v=vs.85).aspx](https://msdn.microsoft.com/en-us/library/windows/desktop/dd318453(v=vs.85).aspx) states that HWINEVENTHOOK is available by simply including windows.h, which we do at the top of this file (and more generally, I would expect that most of the structs/types that we define in this file are available through windows.h so I'd like to understand why our definitions would not be redundant)

Reply:
In the header file it is inside `#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)`. To be safe I redefined it here.

It looks like it is not needed though.